securitycenter/assets/list_project_assets_and_state_changes.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Contains examples calls to Cloud Security Center ListAssets API method.
+package assets
+
+// [START list_project_assets_and_state_changes]
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	securitycenter "cloud.google.com/go/securitycenter/apiv1"
+	"github.com/golang/protobuf/ptypes"
+	"google.golang.org/api/iterator"
+	securitycenterpb "google.golang.org/genproto/googleapis/cloud/securitycenter/v1"
+)
+
+// listAllProjectAssetsAndStateChange lists all current GCP project assets in
+// orgID and prints the projects and there change from a day ago out to w. The
+// method returns the number of project assets found.  orgID is the numeric
+// organization ID of interest.
+func listAllProjectAssetsAndStateChanges(w io.Writer, orgID string) (int, error) {
+	// orgID := "12321311"
+	// Instantiate a context and a security service client to make API calls.
+	ctx := context.Background()
+	client, err := securitycenter.NewClient(ctx)
+	if err != nil {
+		return -1, fmt.Errorf("Error instantiating client %v\n", err)
+	}
+	defer client.Close() // Closing the client safely cleans up background resources.
+	onlyProjects := "security_center_properties.resource_type=" +
+		"\"google.cloud.resourcemanager.Project\""
+
+	daysAgo, _ := time.ParseDuration("24h")
```

Comment:
```suggestion
	daysAgo := 24 * time.Hour
```

https://play.golang.org/p/UP5SmnetaLF